src/ir/var.rs

```diff
@@ -169,3 +165,22 @@ fn parse_int_literal_tokens(cursor: &clang::Cursor,
         lit.parse().ok()
     }
 }
+
+fn get_integer_literal_from_cursor(cursor: &clang::Cursor,
+                                   unit: &clang::TranslationUnit) -> Option<i64> {
+    use clangll::*;
+    let mut value = None;
+    cursor.visit(|c, _| {
+        match c.kind() {
+            CXCursor_IntegerLiteral => {
+                value = parse_int_literal_tokens(&c, unit);
+            }
+            CXCursor_FirstExpr => {
+                value = get_integer_literal_from_cursor(&c, unit);
+            }
+            _ => ()
+        }
+        CXChildVisit_Continue
```

Comment:
Also, probably you want something here like `if value.is_some() { CXChildVisit_Break } else { CXChildVisit_Continue }`.